suitesparse_graphblas/__init__.py

```diff
@@ -37,8 +43,132 @@ def initialize(*, blocking=False, memory_manager="numpy"):
     elif memory_manager == "c":
         lib.GrB_init(blocking)
     else:
-        raise ValueError(f'memory_manager argument must be "numpy" or "c"; got: {memory_manager!r}')
+        raise ValueError(
+            f'memory_manager argument must be "numpy" or "c"; got: ' "{memory_manager!r}"  # noqa
```

Comment:
Why was this changed?  `"{memory_manage!r}"` is no longer in the f-string and won't render the argument.